trikot-viewmodels-declarative-flow/swift/swiftui/Components/VMDButton.swift

```diff
@@ -16,12 +18,26 @@ public struct VMDButton<Label, Content: VMDContent>: View where Label: View {
     }
 
     public var body: some View {
-        Button(action: {
-            self.viewModel.actionBlock()
-        }, label: {
+        Button {
+            if let externalActionHandler = externalActionHandler {
+                externalActionHandler {
+                    viewModel.actionBlock()
+                }
+            } else {
+                viewModel.actionBlock()
+            }
+        } label: {
             labelBuilder(viewModel.content)
-        })
-            .disabled(!viewModel.isEnabled)
-            .hidden(viewModel.isHidden)
+        }
+        .disabled(!viewModel.isEnabled)
+        .hidden(viewModel.isHidden)
+    }
+}
+
+extension VMDButton {
```

Comment:
Maybe move this into its own `VMButton+Configuration` file? Just to make sure to be consistent with other components.